websocket.go

```diff
@@ -0,0 +1,59 @@
+package main
```

Comment:
We may have a server here for testing, but I'd rather have something simple that does tcp over websocket.